docs/front-matter.md

```diff
@@ -194,4 +196,8 @@ relatedlinks:
     - url: /link4
       title: Link 4
       description:
-```
\ No newline at end of file
+```
+
+### Production flag
+
+Metadata to indicate whether page should be included in production build and deploy pipeline. Defaults to `true` if absent. Useful for creating one-time prototype pages that should only be depoloyed on staging (e.g. this [sandbox for an appeals tool](https://github.com/department-of-veterans-affairs/vagov-content/blob/d5ce4f9ce0c3fc178a9856512829667933cff2df/pages/dst/index.md)).
```

Comment:
Can we add a link to our main doc on feature fags? https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Engineering/Front%20End%20Feature%20Flags.md

Reply:
👍